[2024-04-26T14:02:22.746+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-26T14:02:22.792+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres.create_postgres_table scheduled__2024-04-16T00:00:00+00:00 [queued]>
[2024-04-26T14:02:22.807+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres.create_postgres_table scheduled__2024-04-16T00:00:00+00:00 [queued]>
[2024-04-26T14:02:22.808+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-26T14:02:22.831+0000] {taskinstance.py:2327} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2024-04-16 00:00:00+00:00
[2024-04-26T14:02:22.840+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=536) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-26T14:02:22.839+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres', 'create_postgres_table', 'scheduled__2024-04-16T00:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmp_lfjh0cz']
[2024-04-26T14:02:22.842+0000] {standard_task_runner.py:63} INFO - Started process 540 to run task
[2024-04-26T14:02:22.842+0000] {standard_task_runner.py:91} INFO - Job 62: Subtask create_postgres_table
[2024-04-26T14:02:22.919+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres.create_postgres_table scheduled__2024-04-16T00:00:00+00:00 [running]> on host 0ceb125076fc
[2024-04-26T14:02:23.075+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stan' AIRFLOW_CTX_DAG_ID='dag_with_postgres' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-04-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-16T00:00:00+00:00'
[2024-04-26T14:02:23.077+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-26T14:02:23.109+0000] {sql.py:276} INFO - Executing: 
            CREATE TABLE if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
        
[2024-04-26T14:02:23.126+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-26T14:02:23.138+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-26T14:02:23.155+0000] {sql.py:457} INFO - Running statement: 
            CREATE TABLE if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
        , parameters: None
[2024-04-26T14:02:23.165+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:407 AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
[2024-04-26T14:02:23.169+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-26T14:02:23.189+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres, task_id=create_postgres_table, execution_date=20240416T000000, start_date=20240426T140222, end_date=20240426T140223
[2024-04-26T14:02:23.339+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-26T14:02:23.373+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-26T14:02:23.376+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
